### PR TITLE
update angular sdk to support 1.0.0 features

### PR DESCRIPTION
**Pull request by yding03**
Adds modal support, modal close support

Removes deprecated features 
- Client-side template seeding
- Client-side email seeding
- Client-side phone seeding
- Client-side client_id seeding
- Client-side passing of authorization_code within onComplete (through the TypeScript definitions)

### CODE REVIEW

**Comment by yding03** [comment]
Considering upgrading major version of Angular to resolve new version compatibility/security issues

File: projects/berbix/src/lib/berbix.component.ts [hunk 1]
```diff
@@ -1,153 +1,195 @@
...
-      options.push('template=' + template);
-    }
-    if (email) {
-      options.push('email=' + encodeURIComponent(email));
-    }
-    if (phone) {
-      options.push('phone=' + encodeURIComponent(phone));
-    }
+    const token = clientToken;
+    var options = ["sdk=BerbixAngular-" + SDK_VERSION];
```
Comment (horaceko) [inline projects/berbix/src/lib/berbix.component.ts:137]: Can we switch these to use template strings?

File: projects/berbix/src/lib/berbix.component.ts [hunk 2]
```diff
@@ -1,153 +1,195 @@
...
-    if (template) {
-      options.push('template=' + template);
-    }
-    if (email) {
-      options.push('email=' + encodeURIComponent(email));
-    }
-    if (phone) {
-      options.push('phone=' + encodeURIComponent(phone));
-    }
+    const token = clientToken;
```
Comment (horaceko) [inline projects/berbix/src/lib/berbix.component.ts:136]: Probably don't need this local var; just use `clientToken` directly.

File: projects/berbix/src/lib/berbix.component.ts [hunk 3]
```diff
@@ -1,153 +1,195 @@
...
+    var options = ["sdk=BerbixAngular-" + SDK_VERSION];
     if (token) {
-      options.push('client_token=' + token);
+      options.push("client_token=" + token);
     }
+    if (showInModal) {
+      if (!showCloseModalButton) {
```
Comment (horaceko) [inline projects/berbix/src/lib/berbix.component.ts:142]: For readability, it's preferred to use the affirmative in the main condition of an if-else, i.e. `if(showCloseModalButton)`.

**Review by horaceko — APPROVED** [review]
I wasn't able to integration test this, but at a high level, this looks good and mirrors the implementation of our other client-side SDKs. Thanks! 👍

**Pull request closed without merging**